README.md

```diff
@@ -1,2 +1,356 @@
+
+- **Welcome message including dates and venue**
+- **Programs that will transpire in the Tech symposium**
+- **Information about experts that will facilitate the symposium**
+- **List of partners**
+- **A showcase of past symposiums**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://dd-obua.github.io/Mic-Portal-Captone/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+- Click this [link](https://github.com/dd-obua/Mic-Portal-Captone) to open the project on GitHub.
+- Click on the `Code` button to copy the clone link
+- Use `git clone git@github.com:dd-obua/Mic-Portal-Captone.git` command to clone the project.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Any modern web browser.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Go inside your folder using `cd your-folder`
+- Use `git clone git@github.com:dd-obua/Mic-Portal-Captone.git` to clone it in your directory.
+
+### Install
+
+Install this project with:
+
+- Use `npm install` to install node modules.
+
+### Usage
+
+To run the project, execute the following command:
+
+- Open index.html file with live-server
+
+### Run tests
+
+To run tests, run the following command:
+
+- Use `npx hint .` for HTML
+- Use `npx stylelint "**/*.{css,scss}"` for CSS
+- Uses `npx eslint .` for JavaScript
+
+### Deployment
+
+You can deploy this project using:
+
+- GitHub Pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: [dd-obua](https://github.com/dd-obua/)
+- Twitter: [@DenisDanielObu1](https://twitter.com/DenisDanielObu1)
+- LinkedIn: [denis-daniel-obua-99024a229](https://www.linkedin.com/in/denis-daniel-obua-99024a229/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Create registration system for those to attend]**
+- [ ] **Create other pages (e.g. News ...)**
+- [ ] **Add more partners**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, you can offer support in terms of feedback, suggestions and finances.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank you to all my learning partners, morning session members and standup team members from Block 1 till now. You have all, in away contributed to this project.
```

Comment:
- [ ] [REQUIRED] Good job so far! The `README` contains an `"Acknowledgements"` section that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) 🙏🏼